src/models/SqlOutputContentProvider.ts

```diff
@@ -294,8 +294,11 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             queryRunner.eventEmitter.on('resultSet', (resultSet) => {
                 this._service.broadcast(resultsUri, 'resultSet', resultSet);
             });
-            queryRunner.eventEmitter.on('batch', (batch) => {
-                this._service.broadcast(resultsUri, 'batch', batch);
+            queryRunner.eventEmitter.on('batchStart', (batch) => {
```

Comment:
I agree, but since this is a larger change, and the hope was to have this all checked in soon for the Dec release, I'll hold off on this until I get going on part 3.